web/src/components/MemoEditor.tsx

```diff
@@ -169,6 +170,29 @@ const MemoEditor = () => {
       editorRef.current.insertText(" ".repeat(TAB_SPACE_WIDTH));
       return;
     }
+
+    for (const symbol of pairSymbols) {
+      if (event.key === symbol[0]) {
+        event.preventDefault();
+        editorRef.current.insertText("", symbol[0], symbol[1]);
+        return;
+      }
+    }
+
+    if (event.key === "Backspace") {
+      if (!editorRef.current) {
+        return;
+      }
```

Comment:
```suggestion
```

Reply:
What does this do/change?

Reply:
What does this do/change?

Reply:
There already has a checker in the beginning. https://github.com/usememos/memos/pull/791/files#diff-e9a053aeb57009e1c1d9fe8f830f4fdd23956ec4fc62446159d07fdd06649272R101